### ISSUE TITLE
Gradient at node is with respect to what?

### ISSUE THREAD

**Issue by abhigenie92** [jcjohnson/pytorch-examples#8]
The backward function receives the gradient of the output Tensors with respect to some scalar value.
What is this scalar value? What does it represent in the computational graph?

**Comment by hunkim** [comment]
Each variable has their own grad after `backward()`. So it's wrt to each variable.

**Comment by jcjohnson** [comment]
The call `loss.backward()` computes the gradient of the loss with respect to each node in the graph.
